src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -900,7 +900,7 @@ def _type_to_property_name(key):
     return type_dict[key.lower()]
 
 
-def export_zone(cmd, resource_group_name, zone_name):
+def export_zone(cmd, resource_group_name, zone_name, file_name=None):
```

Comment:
do we need help description for the newly added argument?

Reply:
The help text is "Path to the DNS zone file to save."